src/vm/i386/asmhelpers.S

```diff
@@ -1162,20 +1162,30 @@ NESTED_END ResolveWorkerChainLookupAsmStub, _TEXT
 NESTED_ENTRY BackPatchWorkerAsmStub, _TEXT, NoHandler
     PROLOG_BEG
     PROLOG_PUSH eax // it may contain siteAddrForRegisterIndirect
+    PROLOG_PUSH ebx
     PROLOG_PUSH ecx
     PROLOG_PUSH edx
     PROLOG_END
 
-    sub     esp, 4     //  for 16 bytes align
+    // Align Stack Frame
+    mov     ebx, esp
+    and     esp, -16
+
+    // Call BackPatchWorkerStaticStub
+    sub     esp, 8     //  16-bytes align padding
```

Comment:
Could you let me know who can jump into this stub? I also think that fixed stack adjustment is enough, but it turns out that there is some call that jumps into this stub with unaligned stack frame (which results in several regressions).

This is why I introduced this complicated adjustment.